server/app.js

```diff
@@ -441,6 +442,10 @@ module.exports = function createApp({
     '/hdc/licencesRequiringComAssignment',
     secureRoute(licencesRequiringComAssignmentRouter(licenceSearchService, audit))
   )
+  app.use(
+    '/hdc/comAssignedLicencesForHandover',
+    secureRoute(comAssignedLicencesForHandoverRouter(licenceSearchService, audit))
```

Comment:
this should sit with admin routes above, the other admin report has a path: `'/admin/downloadCasesWithCOM/',`

Reply:
should probably sit in `/admin` routes folder too

Reply:
This will also restrict the route so only BATCHLOAD can access it. 
`/server/routes/config/authorisation.js`

Reply:
We're probably missing an entry in `authorisation.js` to limit the CA report to only CAs.

Reply:
Oh but this report is the other CA one, the new admin report is to follow in the next ticket?

Reply:
Limited both CA reports to only CAs in `authorisation.js`